docs/development/codeguide.rst

```diff
@@ -143,11 +143,11 @@ Coding Style/Conventions
   maximum line length for different subpackages (typically either 80 or 100
   characters).  Please try to maintain the style when adding or modifying code.
 
-* One exception is to be made from the PEP8 style: new style relative imports
-  of the form ``from . import modname`` are allowed and required for Astropy,
-  as opposed to absolute (as PEP8 suggests) or the simpler ``import modname``
-  syntax. This is primarily due to improved relative import support since PEP8
-  was developed, and to simplify the process of moving modules.
+* One exception is to be made from the PEP8 style: while absolute imports
```

Comment:
This isn't really an exception, right? PEP8 says
> However, explicit relative imports are an acceptable alternative to absolute imports, especially when dealing with complex package layouts where using absolute imports would be unnecessarily verbose

I'll make a PR suggesting a re-word